contrib/pkg_tester/test_fdb_pkgs.py

```diff
@@ -103,6 +105,8 @@ def ubuntu_image_with_fdb_helper(versioned: bool) -> Iterator[Optional[Image]]:
         container = Container("ubuntu")
         for deb in debs:
             container.copy_to(deb, "/opt")
+        container.run(["bash", "-c", "apt-get update"])
+        container.run(["bash", "-c", "apt-get install --yes binutils"]) # this is for testing libfdb_c execstack permissions
```

Comment:
My limited understanding of Dockerfiles is that one saves disk space and build time by combining as many commands into a single `run` as possible because that produces fewer "layers" or something like that. Does the Python library already do that magically, or should we `apt-get update && apt-get install` instead here? I might also be completely mistaken, in which case you can just ignore this comment.

Reply:
This isn't using a Dockerfile, but both these commands run before saving the result as a docker image so I think it's already behaving like you're describing.